source/query/queries.txt

```diff
@@ -0,0 +1,27 @@
+.. _past-query-view:
+
+=================
+View Past Queries
+=================
+
+Compass automatically stores up to 20 most recent queries for each
+collection.
+
+To view these queries:
+
+#. Click the :icon:`ellipsis-h` in the query bar
+#. Click :guilabel:`Toggle Query History`.
+
+   .. figure:: /images/compass/query-history-view.png
+
+From the :guilabel:`Past Queries` view, you can view
```

Comment:
Do you think we need any of the content from here to the end of the page? The "View Past Queries" subsection is quite clear.

Reply:
Fixed.

Reply:
So clean! Looks great.